src/hooks/useConnectRelatedVenues.ts

```diff
@@ -0,0 +1,62 @@
+import { siblingVenuesSelector, subvenuesSelector } from "utils/selectors";
+import { useConnectCurrentVenueNG } from "./useConnectCurrentVenueNG";
+import { useSelector } from "./useSelector";
+import {
+  SparkleRFQConfig,
+  useSparkleFirestoreConnect,
+} from "./useSparkleFirestoreConnect";
+
+export const useConnectRelatedVenues = (venueId: string | null) => {
+  const { currentVenue } = useConnectCurrentVenueNG(venueId);
+
+  const subvenuesQuery: SparkleRFQConfig = {
+    collection: "venues",
+    where: [["parentId", "==", venueId]],
+    storeAs: "subvenues",
+  };
+  const subvenues = useSelector(subvenuesSelector);
+  const subvenueEventsQueries: SparkleRFQConfig[] = subvenues?.map(
+    (subvenue) => ({
+      collection: "venues",
+      doc: subvenue.id,
+      subcollections: [{ collection: "events" }],
+      orderBy: ["start_utc_seconds", "asc"],
+      storeAs: "subvenueEvents",
+    })
```

Comment:
Since this is mapped, each individual entry will try and overwrite the same `subvenueEvents` won't it?

Reply:
I thought they merged into the same place?

Reply:
- http://react-redux-firebase.com/docs/queries#storeAs
  - > Data is stored in redux under the path of the query for convenience. This means that two different queries to the same path (i.e. todos) will both place data into state.data.todos **even if their query parameters are different.**